benchmarks/pegasus_bench.py

```diff
@@ -79,7 +79,8 @@ def benchmark_sv(self, pto, scheme):
         th.update(
             {
                 "PTO": [pto],
-                "fact_to_ren_scale_ratio": [np.sqrt(0.5), np.sqrt(2.0)],
+                # TODO: why do we need this epsilon to have a good match ??
+                "fact_to_ren_scale_ratio": [np.sqrt(0.5),np.sqrt(2.0-1e-8)],
```

Comment:
Good point.

If you find by yourself that's perfect, but maybe we should have a look together.

Reply:
Indeed it was the same trick needed for the LHA tables. Now bench with `scheme a` (the only one available in pegasus) it's perfect!

Reply:
Very good, thank you :D